src/store/collection.ts

```diff
@@ -80,6 +92,16 @@ export class AppCollection {
         let tree = await this.updateTree()
         
         const c = await getAppCommunity()
-        return c.playTransactions(tree, [setDataTransaction(`trackables/${trackable.did}`, true)])
+        return c.playTransactions(tree, [setDataTransaction(`trackables/${trackable.did}`, false)]) // false means "unowned"
+    }
+
+    async ownTrackable(trackable: Trackable, user: User) {
```

Comment:
I get skipping this step as it doesn't add any value at the moment, but it does seem odd that the driver doesn't do a "setOwnership" on the `Trackable` chaintree

Reply:
yup! but right now there's no ownership model to do that.